content/source/docs/enterprise/api/users.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: enterprise2
+page_title: "Users - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-users"
+---
+
+# Users API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## Show a User
+
+`GET /users/:user_id`
```

Comment:
Since there's no list endpoint, we could use a short sentence here about where you would usually expect to get these user IDs (I'm assuming it's stuff like the team members endpoints?). Likewise, do you think we should link TO this page from any of the endpoints that return user IDs? 

Otherwise this all looks good!

Reply:
I've added a note about the user ID. Thanks!

Since there are many endpoints that return user IDs I created a task to link to this user details endpoint.